app/models/episode_story_handler.rb

```diff
@@ -54,7 +54,7 @@ def update_attributes
     episode.summary = sa[:description]
     episode.content = sa[:description]
     episode.categories = sa[:tags]
-    episode.published_at = Time.parse(sa[:published_at]) if sa[:published_at]
+    episode.published_at = sa[:published_at] ? Time.parse(sa[:published_at]) : nil
```

Comment:
added this so that there would be a way to set `episode.published_at` to `nil` -- otherwise it kept retaining its previous value and staying published. Is the unpublish action / the setting of `published_at` to `nil` actually happening elsewhere in the code? I assume so but can't find it...